### PR TITLE
Move gem install section to main part.

### PR DESCRIPTION
**Pull request by simi**
implementing https://github.com/rubygems/rubygems.org/issues/3323

## before

image

## after

image

### CODE REVIEW

File: app/assets/stylesheets/type.css [hunk 1]
```diff
@@ -46,7 +46,7 @@ label.t-hidden {
   text-transform: uppercase;
   color: #e9573f; }
   .t-list__heading:not(:first-child) {
-    margin-top: 25px; }
+    margin-top: 30px; }
```
Comment (simi) [inline app/assets/stylesheets/type.css:49]: 1. usually 30px or 32px is used as a "gutter"
2. for some mysterious reasons 25px makes render problems (you can see some of the at "before" screenshot as well), this fixes misalignment finally

**Review by indirect — APPROVED** [review]
Looks good! Could we put it below the gem description, above the list of versions?

**Comment by simi** [comment]
@indirect updated (including screenshot)

**Comment by indirect** [comment]
Thanks! 🚢

**Comment by fulldecent** [comment]
Screenshot looks good!

**Review by segiddins — APPROVED** [review]

**Review by hsbt — APPROVED** [review]

**Pull request merged**